src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -22,97 +22,98 @@ type BillingStatusResult = {
 function getBillingStatus(translate: LocaleContextProps['translate'], accountData?: AccountData): BillingStatusResult | undefined {
     const cardEnding = (accountData?.cardNumber ?? '')?.slice(-4);
 
-    const amountOwed = SubscriptionUtils.getAmountOwed();
+    const amountOwed = getAmountOwed();
 
-    const subscriptionStatus = SubscriptionUtils.getSubscriptionStatus();
+    const subscriptionStatus = getSubscriptionStatus();
 
-    const endDate = SubscriptionUtils.getOverdueGracePeriodDate();
+    const endDate = getOverdueGracePeriodDate();
 
     const endDateFormatted = endDate ? DateUtils.formatWithUTCTimeZone(fromUnixTime(endDate).toUTCString(), CONST.DATE.MONTH_DAY_YEAR_FORMAT) : null;
 
     const isCurrentCardExpired = DateUtils.isCardExpired(accountData?.cardMonth ?? 0, accountData?.cardYear ?? 0);
 
     switch (subscriptionStatus?.status) {
-        case SubscriptionUtils.PAYMENT_STATUS.POLICY_OWNER_WITH_AMOUNT_OWED:
+        case PAYMENT_STATUS.POLICY_OWNER_WITH_AMOUNT_OWED:
             return {
                 title: translate('subscription.billingBanner.policyOwnerAmountOwed.title'),
                 subtitle: translate('subscription.billingBanner.policyOwnerAmountOwed.subtitle', {date: endDateFormatted ?? ''}),
                 isError: true,
                 isRetryAvailable: true,
             };
 
-        case SubscriptionUtils.PAYMENT_STATUS.POLICY_OWNER_WITH_AMOUNT_OWED_OVERDUE:
+        case PAYMENT_STATUS.POLICY_OWNER_WITH_AMOUNT_OWED_OVERDUE:
             return {
                 title: translate('subscription.billingBanner.policyOwnerAmountOwedOverdue.title'),
                 subtitle: translate('subscription.billingBanner.policyOwnerAmountOwedOverdue.subtitle'),
                 isError: true,
+                isRetryAvailable: true,
```

Comment:
Note this is the only substantive change, the rest is lint fixes/fix for the corresponding test